javacutil/src/main/java/org/checkerframework/javacutil/trees/TreeBuilder.java

```diff
@@ -145,10 +146,21 @@ public MemberSelectTree buildCloseMethodAccess(ExpressionTree autoCloseableExpr)
     // Find the close() method
     Symbol.MethodSymbol closeMethod = null;
 
-    for (ExecutableElement method : ElementFilter.methodsIn(elements.getAllMembers(exprElement))) {
-      if (method.getParameters().isEmpty() && method.getSimpleName().contentEquals("close")) {
-        closeMethod = (Symbol.MethodSymbol) method;
-        break;
+    // We could use elements.getAllMembers(exprElement) to find the close method, but in rare cases
+    // calling that method crashes with a Symbol$CompletionFailure exception.  See
+    // https://github.com/typetools/checker-framework/issues/6396.  The code below directly searches
+    // all supertypes for the method and avoids the crash.
+    Name closeName = names.fromString("close");
```

Comment:
I think this call is potentially costly, which is why the Checker Framework doesn't use `Name`s very often.  So you could try making it a field.